modules/extensions-spring-support/src/test/resources/typed-value-on-config-config.xml

```diff
@@ -6,17 +6,17 @@
                http://www.mulesoft.org/schema/mule/typed-value http://www.mulesoft.org/schema/mule/typed-value/current/mule-typed-value.xsd">
 
 
-    <typed-value:config name="dynamicConfig" stringTypedValue="#[mel:payload]">
-        <typed-value:differed-door address="#[mel:flowVars.address]"/>
+    <typed-value:config name="dynamicConfig" stringTypedValue="#[payload]">
+        <typed-value:differed-door address="#[vars.address]"/>
     </typed-value:config>
 
     <typed-value:config name="staticConfig" stringTypedValue="string">
         <typed-value:differed-door address="address"/>
     </typed-value:config>
 
     <flow name="typedValueOnDynamicConfig">
-        <set-variable variableName="address" value="address"/>
-        <set-payload value="string" mimeType="application/json" encoding="UTF-8"/>
+        <set-variable variableName="address" value="&quot;address&quot;" mimeType="application/json" encoding="UTF-8"/>
```

Comment:
why these quotes? Sounds like a cover up for something that broke

Reply:
nope, is the way to pass an string with mimeType json. Talked about this with stiv, is how typed values work

Reply:
why did you need to change it then?